src/brevitas/core/quant/float.py

```diff
@@ -90,4 +90,4 @@ def forward(self, x):
         y = self.float_clamp_impl(y)
         y = self.dequantize(y, scale)
         # This is to respect the current interface of proxies
-        return y, scale, self.zero_point_impl(), self.bit_width()
+        return y, scale, self.zero_point_impl(), self.exponent_bit_width(), self.mantissa_bit_width()
```

Comment:
We need to merge the FloatQuantTensor PR so that we don't need to rely on this hack